.travis.yml

```diff
@@ -27,14 +24,10 @@ jobs:
       env: CHECK="test unit --puppet-version=6.21.1" # PE 2019.8
       stage: spec
     -
-      env: CHECK="test unit --puppet-version=7.4.1"
```

Comment:
a similar idea would be to move this out of travis:
https://github.com/puppetlabs/support-tasks/blob/main/.github/workflows/spec.yml

the above will test automatically on supported OS and puppet versions and does not require specirfication